setup.py

```diff
@@ -258,6 +258,19 @@ def build_js_files(self):
     undef_macros=UNDEF_MACROS,
 )
 
+MEMRAY_INJECT_EXTENSION = Extension(
```

Comment:
Some observations here from running this on MacOS:

* lldb will fail to load the memray symbol if this is compiled with lto and with optimizations. To make it work I had to deactivate LTO and pass `-Og -g3` to it.

Reply:
We iterated on this a bit: `lldb` can be coerced to work even without DWARF if the symbol we have it call is `extern "C"` and outside the `memray` C++ namespace, and we cast it to the right type before calling it. That's the fix we're going with.